DOCUMENTATION.md

```diff
@@ -230,23 +288,25 @@ an empty line is the end of a chunk.
 . x x
 3 3 3
 
-? x x 
-``` 
+? x x
+```
 
 The empty line splitter will split things this way : `(1, 2)`, `(., 3)`, and `(?)`.
 
-## Validation
+## `<output>`
 
-While XML validation is not a feature of Protogenie, you might be interested in some methods to do that:
+The `<output>` element allows user to control the format of the dev, train and test files.
+If we use the `<default-header>`, it is possible to re-use it the following way:
 
-- You can install `pip install jingtrang` and then run `pyjing path/to/schema.rng path/to/xml`
-- Oxygen XML Editor provides validation while editing. 
-- Probably [XML Mind Editor](https://www.xmlmind.com/xmleditor/download.shtml) does to.
-- Feel free to propose others if you have knowledge we don't
+````xml
+<output column_marker="TAB">
+    <header name="default"/>
+</output>
+```
 
 ## Post-processing
```

Comment:
And here there is a lot of post-processor not documented, once again, my bad :)

Reply:
Your call

Reply:
I'll add them, you'll feel free to enhance my doc :D

Reply:
At your service